### PR TITLE
{lib}[GCC-5.4.0-2.26] lftp 

### PR DESCRIPTION
**Pull request by robqiao**
{lib}[GCC-5.4.0] lftp config

### CODE REVIEW

File: easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    'http://lftp.yar.ru/ftp/',
+    'http://lftp.yar.ru/ftp/old/',
+]
+
+
+dependencies = [
+    ('GnuTLS', '3.5.14', '', ('GCCcore', '5.4.0')),
+    ('gettext', '0.19.8'),
+    ('Autoconf', '2.69', '', True), 
+    ('libreadline', '7.0', '', ('GCCcore', '5.4.0')),
```
Comment (boegel) [inline easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb]: I think you'll need to stick with `6.3` here, due to other dependencies already using that

There's no need to hard specify which toolchain to use here, `GCCcore/5.4.0` will be considered automatically by EasyBuild v3 if no matching easyconfig using `GCC/5.4.0-2.26` is found

Comment (robqiao) [reply easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb]: @boegel do we have a flag that can update easyconfig file to a new toolchain and correct file name correspondingly?

**Comment by robqiao** [comment]
@boegel plz review this pr?

File: easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::     Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::     GNU GPL-v3.0
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb]: if i recall right, EB under GPLv2 so this is potentially a redistribution issue...

File: easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb]: I think you're looking for `--try-toolchain`?

File: easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb]: Yes, indeed, we ship `easybuild-easyconfigs` packages under GPLv2.

On the other hand, there's very little creativity involved with composing easyconfig files, so I'm not sure whether this is really an issue...

@robqiao Can you please reconsider either changing the license to something compatible with GPLv2, or simply dropping the license line?

Comment (boegel) [reply easybuild/easyconfigs/a/Autotools/Autotools-20170619-GCCcore-6.3.0.eb]: To clarify: the license you specify here is assumed to be the license for the easyconfig file, not the license for the software it is installing...

**Comment by robqiao** [comment]
@boegel Since lftp-4.7.8-GCC-6.3.0-2.28.eb is now in the repo, I will now close this PR.

**Comment by boegel** [comment]
Thanks for the follow-up @robqiao!

**Pull request closed without merging**